src/main/java/com/gluonhq/NativeBaseMojo.java

```diff
@@ -96,6 +96,9 @@
     @Parameter(property = "client.jniList")
     List<String> jniList;
 
+    @Parameter(property = "client.compilerArgs")
+    List<String> compilerArgs;
```

Comment:
It should be better renaming this extension to `client.nativeImageArgs`, to avoid a possible confusion to the end developer